.flake8

```diff
@@ -1,8 +1,9 @@
 [flake8]
-ignore = E203, W503
+ignore = E203, W503, C408, SIM117
```

Comment:
C408 is ignored because otherwise it insists that `dict(a=2, b=3)` is rewritten as `{"a": 2, "b":3}`. In long dictionaries the first option is less verbose because you avoid all the double quotes, so we ignore it in all eo-* packages.

SIM117:
```python
with LocalFile(...) as local:
    with open(...) as f:
        ...

# wants it to be changed to
with LocalFile(...) as local, open(...) as f:
    ...
```
This indeed gets rid of identation levels, BUT in eo-grow we have huge rasterio context managers, which span multiple lines. These cannot be merged without disgusting syntax (until 3.10).